### ISSUE TITLE
create demo?

### ISSUE THREAD

**Issue by ryuken** [unindented/react-puzzle#49]
Hi interesting project, is it possible to create a demo site?

**Comment by unindented** [comment]
Yup, I'll try to find time to do it. Right now it's used in this browser extension: https://chrome.google.com/webstore/detail/puzzle-creator/mbalnpbcmecdckpghgacibglihkgamkl

The source for the extension lives here: https://github.com/unindented/puzzle-creator

**Comment by ryuken** [comment]
Cool! Thanks.
